packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -353,7 +353,7 @@ export interface IrisGridState {
 
   conditionalFormats: SidebarFormattingRule[];
   conditionalFormatEditIndex: number | null;
-  conditionalFormatPreview?: SidebarFormattingRule | null;
+  conditionalFormatPreview?: SidebarFormattingRule | undefined;
```

Comment:
This is redundant, the `?` already implies `undefined`:
```suggestion
  conditionalFormatPreview?: SidebarFormattingRule;
```